src/qt/guiutil.cpp

```diff
@@ -764,10 +764,10 @@ QString NetworkToQString(Network net)
     assert(false);
 }
 
-QString ConnectionTypeToQString(ConnectionType conn_type)
+QString ConnectionTypeToQString(ConnectionType conn_type, bool relay_txes)
 {
     switch (conn_type) {
-    case ConnectionType::INBOUND: return QObject::tr("Inbound");
+    case ConnectionType::INBOUND: return relay_txes ? QObject::tr("Inbound Full Relay") : QObject::tr("Inbound Block Relay");
```

Comment:
nit: Is a tooltip update required (see the screenshot)?

Reply:
I should have mentioned that I tried various tooltip updates, like "Inbound Full Relay or Block Relay" and "Inbound Full/Block Relay", but these (a) looked cluttered to me and (b) aren't "official" connection type names yet. So TBH I don't know.

Reply:
Updated to Inbound Full/Block Relay